.github/workflows/ci.yml

```diff
@@ -68,19 +68,21 @@ jobs:
       - name: JavaScript test suite 
         run: |
             yarn
-            yarn build
+            yarn coverage
             yarn run test 
       
       - name: JavaScript linting
         run: yarn lint-non-build
-      
-      - name: Ruby rspec test suite
-        run : COVERAGE=true bundle exec rspec spec/ --color --profile --format documentation
+
+      - name: Test & publish code coverage
```

Comment:
I think the previous name is better; the tests being run in this step are much more important than the coverage.